README.md

```diff
@@ -12,3 +12,42 @@ docker buildx build -t waggle/gpu-stress-test --platform linux/amd64,linux/arm/v
 (`failed to solve: rpc error: code = Unknown desc = failed to copy: unexpected EOF`) when
 building the `linux/arm64` platform. This was resolved by building and pushing on
 a true Ubuntu:18.04 host.
+
+## Running the gpu-stress-test container
+
+### amd64 platform
+
+For the latest nvidia-container-runtime version use
+
+```
+docker run -ti --rm \
+  --gpus all \
+  gpu_stress_test
+```
+
+otherwise use
+
+```
+docker run -ti --rm \
+  --runtime nvidia \
+  gpu_stress_test
+```
+
+### armv7 platform
+
+TBD
+
+### arm64 platform
+
```

Comment:
We'd better specify that we are targeting NVIDIA Jetson devices for this arm64 platform.